doc/api/deprecations.md

```diff
@@ -634,6 +634,17 @@ Type: Runtime
 
 *Note*: change was made while `async_hooks` was an experimental API.
 
+<a id="DEP00XX"></a>
```

Comment:
As far as I see from Git history, deprecations are usually assigned a number immediately in the pull request.